### DISCUSSION TITLE
Does dify have embedded interface documentation, or is there any way to generate interface documentation？plz

### DISCUSSION THREAD

**Discussion by einherjar9527** [langgenius/dify#8238] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I want to do secondary development, need interface documentation, who can provide it ,similar to swagger, plz

**Comment by StrayDragon** [comment]
Hey I had similar needs recently and discovered the Discussions. 
I built a project https://github.com/StrayDragon/dify-openapi , hope it's helpful
If you're interested, feel free to track the issue #14339 in case it maybe accepted by the official team someday :)
